bifrost/release.go

```diff
@@ -236,33 +241,47 @@ func (r *Release) TimedOut() error {
 ///////
 
 // UnlockRootLock deletes the Lock File for the release
-func (r *Release) UnlockRoot(s3c aws.S3API) error {
-	return s3.ReleaseLock(s3c, r.Bucket, r.RootLockPath(), *r.UUID)
+func (r *Release) UnlockRoot(s3c aws.S3API, locker Locker, lockTableName string) error {
+	if err := s3.ReleaseLock(s3c, r.Bucket, r.RootLockPath(), *r.UUID); err != nil {
```

Comment:
Style nit: This line is not primarily a conditional, it's an _unconditional_ call of `ReleaseLock`. So don't hide it behind an `if`.

(Same comment applies elsewhere in this PR.)

```suggestion
	err := s3.ReleaseLock(s3c, r.Bucket, r.RootLockPath(), *r.UUID)
	if err != nil {
```

Reply:
Agree